main.js

```diff
@@ -58,6 +63,13 @@ document.addEventListener('keydown', function (event) {
 
 document.addEventListener('keydown', function (event) {
   if (event.keyCode === 32) {
-    myCar.start()
+    if (!this.isStarted) {
```

Comment:
This event listener function is not part of your car. Check `window.isStarted` in your browser console.